bom/application/pom.xml

```diff
@@ -172,7 +172,7 @@
         <artemis.version>2.11.0</artemis.version>
         <proton-j.version>0.33.8</proton-j.version>
         <okhttp.version>3.14.9</okhttp.version>
-        <sentry.version>1.7.30</sentry.version>
+        <sentry.version>3.2.0</sentry.version>
```

Comment:
Outch. Dependabot rule, maybe? :)

Reply:
@gsmet wdym?

Reply:
That it would probably be a good idea to add it to Dependabot so that it gets updated automatically.

See https://github.com/quarkusio/quarkus/blob/master/.github/dependabot.yml#L12

Reply:
I pushed a commit to add it. We should be good now.

Reply:
Cool, thanks 👍